asset-transfer-private-data/chaincode-go/README.md

```diff
@@ -40,81 +40,24 @@ The test network is deployed with two peer organizations. The `createChannel` fl
-export CORE_PEER_TLS_ENABLED=true
-export CORE_PEER_LOCALMSPID="Org1MSP"
-export CORE_PEER_MSPCONFIGPATH=${PWD}/organizations/peerOrganizations/org1.example.com/users/Admin@org1.example.com/msp
-export CORE_PEER_TLS_ROOTCERT_FILE=${PWD}/organizations/peerOrganizations/org1.example.com/peers/peer0.org1.example.com/tls/ca.crt
-export CORE_PEER_ADDRESS=localhost:7051
-```
-
-Run the following command to package the private asset transfer chaincode:
-```
-peer lifecycle chaincode package private_transfer.tar.gz --path ../asset-transfer-private-data/chaincode-go --lang golang --label private_transfer_1
-```
-
-The command will create a chaincode package named `private_transfer.tar.gz`. We can now install this package on the Org1 peer:
-```
-peer lifecycle chaincode install private_transfer.tar.gz
-```
-
-You will need the chaincode package ID in order to approve the chaincode definition. You can find the package ID by querying your peer:
-```
-peer lifecycle chaincode queryinstalled
-```
-Save the package ID as an environment variable. The package ID will not be the same for all users, so need to use the result that was returned by the previous command:
-```
-export PACKAGE_ID=private_transfer_1:d195682c777705f76a4cdce903a1365dc93a9b5b6fb363eeac292e616cfb64d2
-```
-You can now approve the chaincode as Org1. This command includes a path to the collection definition file.
-```
-peer lifecycle chaincode approveformyorg -o localhost:7050 --ordererTLSHostnameOverride orderer.example.com --channelID mychannel --name private_transfer --version 1 --package-id $PACKAGE_ID --sequence 1 --tls --cafile ${PWD}/organizations/ordererOrganizations/example.com/orderers/orderer.example.com/msp/tlscacerts/tlsca.example.com-cert.pem --collections-config ../asset-transfer-private-data/chaincode-go/collections_config.json --signature-policy "OR('Org1MSP.peer','Org2MSP.peer')"
+You can use the test network script to deploy the private data smart contract to the channel that was just created. Deploy the smart contract to `mychannel` using the following command:
 ```
-
-Note we are approving a chaincode endorsement policy of `"OR('Org1MSP.peer','Org2MSP.peer')"`. This allows Org1 and Org2 to create an asset without receiving an endorsement from the other organization.
-
-
-### Install and approve the chaincode as Org2
-
-We can now install and approve the chaincode as Org2. Set the following environment variables to operate as the Org2 admin:
-```
-export CORE_PEER_TLS_ENABLED=true
-export CORE_PEER_LOCALMSPID="Org2MSP"
-export CORE_PEER_MSPCONFIGPATH=${PWD}/organizations/peerOrganizations/org2.example.com/users/Admin@org2.example.com/msp
-export CORE_PEER_TLS_ROOTCERT_FILE=${PWD}/organizations/peerOrganizations/org2.example.com/peers/peer0.org2.example.com/tls/ca.crt
-export CORE_PEER_ADDRESS=localhost:9051
+./network.sh deployCC -ccn private -ccep "OR('Org1MSP.peer','Org2MSP.peer')" -cccg ../asset-transfer-private-data/chaincode-go/collections_config.json
```

Comment:
Adding these args explicitly will be useful to those modifying & trying out CC deploys:
`-ccs 1 -ccv 1.0`

and how about this additional detail:
`Deploys chaincode with name `private`, version `1.0`, sequence `1`, collection configuration from file `collections_config.json`. `

Reply:
I do not think this detail should be added to this discussion. l think we should keep is simple and to the point of the private data and collection. The collection file is already discussed above and below in the readme.